tests/spec/README.txt

```diff
@@ -0,0 +1,6 @@
+These tests specify the expected behaviour of ehrQL, and by extension, the QM.
```

Comment:
These tests are very verbose, especially compared to the in-memory database tests. Something like the string/table approach might help. I think also we could consider not creating whole datasets in each test since you only ever need a single variable for the language spec (I think).

Reply:
> These tests are very verbose, especially compared to the in-memory database tests. Something like the string/table approach might help.

Yes, I think it would help.

>  I think also we could consider not creating whole datasets in each test since you only ever need a single variable for the language spec

I'll experiment.